frontend/src/lib/FileUploadClient.test.ts

```diff
@@ -83,6 +83,7 @@ describe("FileUploadClient Upload", () => {
         }
 
         if (getCookie("_xsrf")) {
+          // @ts-ignore
           if (!("X-Xsrftoken" in config.headers)) {
             return [403]
           }
```

Comment:
Can you add a comment explaining why the `@ts-ignore` is necessary, so we can remove it if/when it's appropriate?